src/palm-proxy.js

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
```

Comment:
Oops, you probably didn't mean to upload this file, since you already have it marked to skip in .gitignore.